ProfileView/src/[sandbox]/sandbox.ts

```diff
@@ -88,21 +88,22 @@ export class Sandbox extends LitElement {
             style=${`width: ${this.containerWidth}; height: ${this.containerHeight}; overflow: auto;`}
             class="widget-container"
           >
+            <!-- profile-read-token="so=demoassure&sn=sandbox&ss=profile&sp=r&se=2022-06-17T23:47:34.409Z&sk=sandbox&sig=61BLCJ5+vZtOOvut/7khUQyg0N9KlvbPrJWrYa9lf28=" -->
```

Comment:
Any reason to keep this rather than delete all the way?

Reply:
Profile API needs a token with Write access. While we only read data and not write. I need to check with Daniel on this.

Reply:
This Profile API is about to change. We might be needing just a read token.

Reply:
Leaving this comment to stay for now.